client/my-sites/hosting/github-authorize-card/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button, Card } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import CardHeading from 'calypso/components/card-heading/index';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors/index';
+import iconGitHub from './github.svg';
+
+import './style.scss';
+
+export const GithubAuthorizeCard = () => {
+	const siteId = useSelector( getSelectedSiteId );
+	return (
+		<Card className="github-authorize-card">
+			<img className="github-icon" src={ iconGitHub } alt="" />
```

Comment:
I don't think `github-icon` is namespaced enough, I can imagine something else in Calypso using this name too. I was thinking something like `github-authorize-icon`.

Reply:
Ok, will do that.